knife-ec2.gemspec

```diff
@@ -19,10 +19,10 @@ Gem::Specification.new do |s|
   s.add_dependency 'fog',           '~> 1.25.0'
   s.add_dependency 'knife-windows', '>= 0.8.2'
 
-  s.add_development_dependency 'mixlib-config', '~> 2.0'
-  s.add_development_dependency 'chef',          '>= 11.16.2', '< 12'
-  s.add_development_dependency 'rspec',         '~> 2.14'
-  s.add_development_dependency 'rake',          '~> 10.1'
+  s.add_development_dependency 'mixlib-config'
+  s.add_development_dependency 'chef'
+  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rake'
   s.add_development_dependency 'sdoc',          '~> 0.3'
```

Comment:
Per our discussion, it's probably best to still pin these, at least adhering to what we support in chef-dk.  I matched the versions to the 0.7.0 RC of chef-dk as follows:

```ruby
  s.add_development_dependency 'chef',  '~> 12.0', '>= 12.2.1'
  s.add_development_dependency 'rspec', '~> 3.0'
  s.add_development_dependency 'rake',  '~> 10.1'
  s.add_development_dependency 'sdoc',  '~> 0.3'
```

rspec tests pass, and running actual commands against AWS succeeded.  I'd recommend that we adopt these unless there's a reason not to.  What say you, @juliandunn? :smile: